flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -872,7 +860,7 @@ public void testKryoRegisteringRestoreResilienceWithRegisteredType() throws Exce
      *
      * @throws Exception expects {@link ExpectedKryoTestException} to be thrown.
      */
-    @Test
+    @TestTemplate
     @SuppressWarnings("unchecked")
     public void testKryoRegisteringRestoreResilienceWithDefaultSerializer() throws Exception {
```

Comment:
```suggestion
    void testKryoRegisteringRestoreResilienceWithDefaultSerializer() throws Exception {
```
could be package private